src/account.c

```diff
@@ -945,6 +978,19 @@ const char *account_mwi(const struct account *acc)
 }
 
 
+/**
+ * Get call transfer capability of an account
+ *
+ * @param acc User-Agent account
+ *
+ * @return "yes" or "no"
+ */
+const char *account_call_transfer(const struct account *acc)
+{
```

Comment:
the input argument (acc) must be checked for NULL.

if NULL you can return either NULL or the string "no".